### PR TITLE
Add menu items manager0002 test

### PR DESCRIPTION
**Pull request by kshitijSharma2014**
I have added 4 tests in it

### CODE REVIEW

**Comment by kshitijSharma2014** [comment]
@puneet0191  @javigomez  @gunjanpatel   please have a look at it.

I have tried to take care of the coding style but still I am afraid If I have not followed it anywhere.
please let me know about any of my mistakes so that I can make changes accordingly.

thank you

**Comment by kshitijSharma2014** [comment]
@puneet0191  @javigomez  @gunjanpatel 
I have two other tests written 
link  https://github.com/kshitijSharma2014/testing-scripts-gsoc/blob/master/problematictests.php

the error comes when I try to add a menu item that is when addMenuItem() is called.

File: tests/system/webdriver/tests/menus/MenuItemsManager0002Test.php [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+use SeleniumClient\By;
+
+/**
+ * This class tests the Article: Front End and Add/Edit Screens.
+ *
+ * @package    Joomla.Test
+ * @subpackage Webdriver
+ * @since      3.2
+ */
+class MenuItemsManager0002Test extends JoomlaWebdriverTestCase{
```
Comment (javigomez) [inline tests/system/webdriver/tests/menus/MenuItemsManager0002Test.php]: braces should start in it's own line. Check http://joomla.github.io/coding-standards/?coding-standards/chapters/php.md.

Please install the sniffer.

File: tests/system/webdriver/tests/menus/MenuItemsManager0002Test.php [hunk 2]
```diff
@@ -0,0 +1,377 @@
...
+	 * @since 3.0
+	 */
+	public function tearDown()
+	{
+		$this->doAdminLogout();
+		parent::tearDown();
+	}
+	
+	 /**
+	 * @test
```
Comment (javigomez) [inline tests/system/webdriver/tests/menus/MenuItemsManager0002Test.php]: missing DocBlock

**Comment by kshitijSharma2014** [comment]
I will add the docblock by tomorrow.

File: tests/system/webdriver/tests/menus/MenuItemsManager0002Test.php (hunk 1 above)
Comment (kshitijSharma2014) [inline tests/system/webdriver/tests/menus/MenuItemsManager0002Test.php]: I have installed the joomla php code sniffer, it was giving me an error here but I couldn't get it why.
I got to know it now. I will make the change, will take care in future pulls

thanks :)

**Comment by kshitijSharma2014** [comment]
I have added a line of description before each line

**Pull request merged**